src/metabase/api/user.clj

```diff
@@ -115,14 +115,14 @@
                                               [:= :core_user.id :permissions_group_membership.user_id])
         (some? group_id) (hh/merge-where [:= :group_id group_id])))
 
-
 (api/defendpoint GET "/"
   "Fetch a list of `Users`. By default returns every active user but only active users.
 
-  If `status` is `deactivated`, include deactivated users only.
-  If `status` is `all`, include all users (active and inactive).
-  Also supports `include_deactivated`, which if true, is equivalent to `status=all`.
-  `status` and `included_deactivated` requires superuser permissions.
+   - If `status` is `deactivated`, include deactivated users only.
+   - If `status` is `all`, include all users (active and inactive).
+   - Also supports `include_deactivated`, which if true, is equivalent to `status=all`.
```

Comment:
maybe clarify these two points as well?
* if both params are passed, `status` takes precedence
* if `include_deactivated` is false, it is equivalent to `status=active`

Reply:
yep, I added these 2 points in